myapp/views.py

```diff
@@ -80,3 +81,9 @@ class BlogList(ListView):
 #List of all Authoer detils 
 class AuthorList(ListView):
     queryset = Author.objects.all()
+
+
+#Blog details
+class BlogDetails(DetailView):
```

Comment:
Change `BlogDetails` to `BlogDetails`. Same for others if you have given the name like this.

Reply:
Rename this to `BlogDetail`

Reply:
After this you can merge the PR.